Develop an SQL schema for SOPs

```diff
@@ -1,29 +1,4351 @@
+    charge integer NOT NULL,
+    theoretical_mass_to_charge double precision NOT NULL,
+    spectra_ref text[] NOT NULL,
+    identification_method text NOT NULL,
+    ms_level text NOT NULL,
+    id_confidence_measure_number text,
+    id_confidence_measure double precision,
+    rank integer NOT NULL
+);
+
+
+ALTER TABLE "MzTab-M"."Small Molecule Evidence (SME) Section" OWNER TO postgres;
+
+--
+-- TOC entry 227 (class 1259 OID 18224)
+-- Name: Small Molecule Evidence (SME) Section_SME_ID_seq; Type: SEQUENCE; Schema: MzTab-M; Owner: postgres
+--
+
+CREATE SEQUENCE "MzTab-M"."Small Molecule Evidence (SME) Section_SME_ID_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE "MzTab-M"."Small Molecule Evidence (SME) Section_SME_ID_seq" OWNER TO postgres;
+
+--
+-- TOC entry 2970 (class 0 OID 0)
+-- Dependencies: 227
+-- Name: Small Molecule Evidence (SME) Section_SME_ID_seq; Type: SEQUENCE OWNED BY; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER SEQUENCE "MzTab-M"."Small Molecule Evidence (SME) Section_SME_ID_seq" OWNED BY "MzTab-M"."Small Molecule Evidence (SME) Section"."SME_ID";
+
+
+--
+-- TOC entry 230 (class 1259 OID 18237)
+-- Name: Small Molecule Feature (SMF) Section; Type: TABLE; Schema: MzTab-M; Owner: postgres
+--
+
+CREATE TABLE "MzTab-M"."Small Molecule Feature (SMF) Section" (
+    "SMF_ID" integer NOT NULL,
+    "SME_ID_REFS" integer,
+    "SME_ID_REF_ambiguity_code" integer,
+    adduct_ion text,
+    isotopomer text,
+    exp_mass_to_charge double precision NOT NULL,
+    charge integer NOT NULL,
+    retention_time_in_seconds double precision,
+    retention_time_in_seconds_start double precision,
+    retention_time_in_seconds_end double precision,
+    abundance_assay_number text,
+    abundance_assay text
+);
+
+
+ALTER TABLE "MzTab-M"."Small Molecule Feature (SMF) Section" OWNER TO postgres;
+
+--
+-- TOC entry 229 (class 1259 OID 18235)
+-- Name: Small Molecule Feature (SMF) Section_SMF_ID_seq; Type: SEQUENCE; Schema: MzTab-M; Owner: postgres
+--
+
+CREATE SEQUENCE "MzTab-M"."Small Molecule Feature (SMF) Section_SMF_ID_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE "MzTab-M"."Small Molecule Feature (SMF) Section_SMF_ID_seq" OWNER TO postgres;
+
+--
+-- TOC entry 2971 (class 0 OID 0)
+-- Dependencies: 229
+-- Name: Small Molecule Feature (SMF) Section_SMF_ID_seq; Type: SEQUENCE OWNED BY; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER SEQUENCE "MzTab-M"."Small Molecule Feature (SMF) Section_SMF_ID_seq" OWNED BY "MzTab-M"."Small Molecule Feature (SMF) Section"."SMF_ID";
+
+
+--
+-- TOC entry 207 (class 1259 OID 17208)
+-- Name: Small Molecule Section; Type: TABLE; Schema: MzTab-M; Owner: postgres
+--
+
+CREATE TABLE "MzTab-M"."Small Molecule Section" (
+    "SMF_ID_REFS" integer,
+    database_identifier text[],
+    chemical_formula text[],
```

Comment:
Should this table be broken down further to avoid the need for array types?

Reply:
done